website/src/utils/use-hash-link.js

```diff
@@ -0,0 +1,10 @@
+export default function useHashLink() {
+  if (window.location.hash) {
+    const hashLink = document.getElementById(
+      window.location.hash.replace("#", "")
+    );
+    if (hashLink) {
+      hashLink.scrollIntoView();
```

Comment:
TIL this exists 👀

Reply:
thought the same thing! like wow this made it such an elegant and native solution. veryyy nice